plumbing/ntwk/ntwk.go

```diff
@@ -13,20 +14,33 @@ type Network struct {
 	host host.Host
 	*pubsub.Subscriber
 	*pubsub.Publisher
+	metrics.Reporter
```

Comment:
BLOCKING: You should sync up with @anorth about network wrappers before merging this.  I just tried something similar with the router and he hit me with [this comment](https://github.com/filecoin-project/go-filecoin/pull/2158#discussion_r262235741).  It might be that this object is simple enough that you can avoid wrapping but I'm not aligned enough with these criteria to make a call.

Reply:
@ZenGround0 thanks for pinging me. @rosalinekarr consider this un-blocked.

Since this is only being used for a command, rather than any functional integration, and there are no tests (not needed), I think it's pragmatic to leave this as is for now. We should aim to abstract libp2p away from most places, this one doesn't seem worth the trouble today.